### ISSUE TITLE
Nextcloud Beta appearance improvement

### ISSUE THREAD

**Issue by OlivierS1** [nextcloud/android#508]
### Actual behaviour
-Tell us what happens
I have jumped into the Beta train and I have noticed the only distinction between normal Nextcloud app and Nextcloud app Beta is the actual name in the main app tab.
When both app are displayed, it because difficult to identify which is which.

### Expected behaviour
-Tell us what should happen
I think the Beta version should have either a different color (Theming may not work properly for that), or some other characteristic made unique just for the Beta version.
each page or menu should be made clearly distinctive.
 
### Steps to reproduce
1. 
2. 
3. 

### Environment data
Android version:

Device model: 

Stock or customized system:

Nextcloud app version:

Nextcloud server version:

#### Web server error log
```
Insert your webserver log here
```

#### Nextcloud log (data/nextcloud.log)
```
Insert your Nextcloud log here
```

**Comment by tobiasKaminsky** [comment]
@eppfel @jancborchardt @AndyScherzinger

**Comment by eppfel** [comment]
I would argue against visually distinguishing the beta. We have the name and the icon.
Visually changing the beta inside the app will decrease maintainability and testing includes the UI, so this would make it also harder to test.
When their would be a theming option, it should be optional, so users can for example use a custom color in the stable and the default blue in the beta.
But I'm curios @OlivierS1 , how do you use both apps in parallel. Do you test the beta a little bit and than go back to the stable for regular use?

**Comment by eppfel** [comment]
Thanks for this detailed answer. From your use-case the option I see, would be a small "beta-flag" which is always top in every view and clearly is not part of the app. 

![beta_flag](https://cloud.githubusercontent.com/assets/1899308/21743265/dbfc232a-d4fe-11e6-9d88-c2797060b613.png)

Is something like these feasable? @tobiasKaminsky @AndyScherzinger 

Other ideas?

**Comment by AndyScherzinger** [comment]
Hi @eppfel that might be a bit tricky but since we moved to using the toolbar in past we should be able to add a small badge to the action bar, if that would be feasible.

**Comment by jancborchardt** [comment]
The name and icon distinguishes the app properly and very visible in the switcher. Adding other things would change the experience too much from the normal app, which also hinders testing. :/
